code/controllers/subsystem/mapping.dm

```diff
@@ -188,6 +189,13 @@ Used by the AI doomsday and the self-destruct nuke.
 		var/turf/open/floor/circuit/C = N
 		C.update_appearance()
 
+/datum/controller/subsystem/mapping/proc/determine_fake_sale()
+	for(var/z in SSmapping.levels_by_trait(ZTRAIT_STATION))
+		if(SSmapping.level_trait(z, ZTRAIT_NOPARALLAX))
```

Comment:
If you want to be really based, make something like `levels_by_any_trait` but that requires all passed traits